src/App.tsx

```diff
@@ -1,14 +1,22 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useContext, useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { CurrentTodo } from './contexts/CurrentTodoProvider';
 
 export const App: React.FC = () => {
+  const [loader, setLoader] = useState(true);
+  const { todo } = useContext(CurrentTodo);
+
+  useEffect(() => {
+    setTimeout(() => setLoader(false), 1000);
+  }, []);
+
```

Comment:
looks strange, you need to show the loader while loading the todos, so please rewrite it